AGILE/Interpreter.cs

```diff
@@ -177,7 +170,14 @@ public void Tick()
                 bool soundStatus = state.Flags[Defines.SOUNDON];
 
                 // Continue scanning LOGIC 0 while the return value is above 0, indicating a room change.
```

Comment:
can you update this comment? `ExecuteLogic` now returns a boolean and not an integer

Reply:
Updated and pushed to branch.